js/src/util/is-testnet.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+export default (chain) => {
+  return chain === 'morden' || chain === 'ropsten' || chain === 'testnet';
```

Comment:
? Redux has isTest, don't want to maintain it in yet another place, we just simplified and got it down to 1. (And had to make changes.)

In addition, we are really starting to clutter /utils now with all these single function files. There is no reason why things can't be grouped, e.g. had to move nullable to protypes once we added another one. (General grumble)